doc/releases/changelog-0.19.0.md

```diff
@@ -0,0 +1,1142 @@
+        gradient_fn=qml.gradients.param_shift,
+        interface="autograd"
+      )
+      return (result[0] + result[1][0, 0])[0]
+
+  res = jax.grad(cost_fn)(params)
+  ```
+
+<h4>New operations and templates</h4>
+
+* Added a new operation `OrbitalRotation`, which implements the spin-adapted spatial orbital rotation gate.
+  [(#1665)](https://github.com/PennyLaneAI/pennylane/pull/1665)
+
+  An example circuit that uses `OrbitalRotation` operation is:
+
+  ```python
+  dev = qml.device('default.qubit', wires=4)
+
+  @qml.qnode(dev)
+  def circuit(phi):
+      qml.BasisState(np.array([1, 1, 0, 0]), wires=[0, 1, 2, 3])
+      qml.OrbitalRotation(phi, wires=[0, 1, 2, 3])
+      return qml.state()
+  ```
+
+  If we run this circuit, we will get the following output
+
+  ```pycon
+  >>> circuit(0.1)
+  array([ 0.        +0.j,  0.        +0.j,  0.        +0.j,
+          0.00249792+0.j,  0.        +0.j,  0.        +0.j,
+          -0.04991671+0.j,  0.        +0.j,  0.        +0.j,
+          -0.04991671+0.j,  0.        +0.j,  0.        +0.j,
+          0.99750208+0.j,  0.        +0.j,  0.        +0.j,
+          0.        +0.j])
+  ```
+
+* Added a new template `GateFabric`, which implements a local, expressive, quantum-number-preserving
+  ansatz proposed by Anselmetti *et al.* in [arXiv:2104.05692](https://arxiv.org/abs/2104.05695).
+  [(#1687)](https://github.com/PennyLaneAI/pennylane/pull/1687)
+
+  An example of a circuit using `GateFabric` template is:
+
+  ```python
+  coordinates = np.array([0.0, 0.0, -0.6614, 0.0, 0.0, 0.6614])
+  H, qubits = qml.qchem.molecular_hamiltonian(["H", "H"], coordinates)
+  ref_state = qml.qchem.hf_state(electrons=2, orbitals=qubits)
+
+  dev = qml.device('default.qubit', wires=qubits)
+
+  @qml.qnode(dev)
+  def ansatz(weights):
+      qml.templates.GateFabric(weights, wires=[0,1,2,3],
+                                  init_state=ref_state, include_pi=True)
+      return qml.expval(H)
+  ```
+
+  For more details, see the [GateFabric documentation](../code/api/pennylane.templates.layers.GateFabric.html).
+
+* Added a new template `kUpCCGSD`, which implements a unitary coupled cluster ansatz with
+  generalized singles and pair doubles excitation operators, proposed by Joonho Lee *et al.*
+  in [arXiv:1810.02327](https://arxiv.org/abs/1810.02327).
+  [(#1743)](https://github.com/PennyLaneAI/pennylane/pull/1743)
+
+  An example of a circuit using `kUpCCGSD` template is:
+
+  ```python
+  coordinates = np.array([0.0, 0.0, -0.6614, 0.0, 0.0, 0.6614])
+  H, qubits = qml.qchem.molecular_hamiltonian(["H", "H"], coordinates)
+  ref_state = qml.qchem.hf_state(electrons=2, orbitals=qubits)
+
+  dev = qml.device('default.qubit', wires=qubits)
+
+  @qml.qnode(dev)
+  def ansatz(weights):
+      qml.templates.kUpCCGSD(weights, wires=[0,1,2,3], k=0, delta_sz=0,
+                                  init_state=ref_state)
+      return qml.expval(H)
+  ```
+
+<h4>Improved utilities across the entire code base</h4>
```

Comment:
Do you think something like?
```suggestion
<h4>Improved utilities for quantum compilation and characterization</h4>
```

Things like device expansion fit into compilation, and the Fourer module fits into characterization :)

Reply:
@antalszava just in case you missed this one :)